code/modules/reagents/reagents.dm

```diff
@@ -48,7 +48,7 @@ GLOBAL_LIST_INIT(name2reagent, build_name2reagent())
 	/// increases as addiction gets worse
 	var/addiction_stage = 0
 	/// does this show up on health analyzers
-	var/scannable = FALSE
+	var/scannable = TRUE
```

Comment:
then scannable is true for everything; so you need to remove from the code.

But I doubt this will go through walance

Reply:
I left it since neuraline seems to want to be non-scannable specifically (For some reason?)
So I left that one alone.
But arguably that one could be allowed to be scanned so idk /shrug